src/Ingredients/DisplayRecipes.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace BreadAndIfit\Ingredients;
+
+
+class DisplayRecipes
+{
+    /**
+     *takes the JSON data returned from the API request and returns a string with HTML markup
+     *
+     * @param $json API response in JSON format
+     * @return string relevant recipes based on user selected ingredients or a error message string
+     */
+    public static function outputRecipes($json)
+    {
+        $recipes = json_decode($json)->results;
```

Comment:
this should be done outside this method. otherwise this method ONLY works with json data

Reply:
Done (thank you...)